src/citrine/resources/table_config.py

```diff
@@ -23,6 +24,11 @@
     IngredientIdentifierInOutput, IngredientQuantityInOutput, \
     IngredientLabelsSetByProcessAndName, IngredientLabelsSetInOutput
 
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:   # pragma: no cover
+    from citrine.resources.project import Project
+    from citrine.resources.team import Team
+
```

Comment:
Are we really benefiting from this?

https://docs.python.org/3.10/library/typing.html?highlight=type_checking#constant

Reply:
It's debatable. This is what allowed me to add the proper type hints on lines 189-190 and 296-297 while still allowing it to compile and without `flake8` complaining. Exactly how valuable that is depends on how much people rely on the type hinting. 🤷